playbooks/rfcs.md

```diff
@@ -39,6 +39,24 @@ Find the right repo:
 - Does it affect the whole dev team? (use [README](https://github.com/artsy/README))
 - Can't figure it out or needs to be private? (use [Potential][] as a fallback)
 
+Create a plan for how the RFC will be implemented. If the change is complex or large in scope consider defining a
+feedback process to iterate on the proposal as you implement and creating milestones and/or metrics to track
+progress.
+
+Decide who needs to know about this proposal, what role teammates will play in the process and how the RFC will be
+considered approved.
+
+- What is the scope of the proposal? Does the whole engineering team need to know or just a smaller group of
+  engineers?
+- Who should actively approve of this RFC? Who should give feedback?
+- Does the change make sense without further discussion from engineering leadership?
+
+Here are some examples of how an RFC could be resolved:
```

Comment:
I like the idea of adding an "implementation" to our RFC playbook, it makes things more tangible. I think the term "resolved" is a bit overloaded, though. To my mind, an RFC moves through the following sequence:

- Proposed (open for discussion)
- Resolved (based on discussion)
- Implemented

As it is, this PR treats "resolved" as a kind of "completed" state, but "resolved" just means that we've decided to do it (even if we haven't done it _yet_). Thinking about how we measure the success of an RFC upfront is a really good idea, but I think we should re-word this a bit to make it more obvious what "resolved" means.

Reply:
Thanks for the feedback, that's a good point that "resolved" is not very clear. My intent here is to address RFC acceptance / approval and be able to clearly answer the question "How do we know we're done discussing?" for a given RFC.

Reply:
Maybe it's better to stick with "approved", for the reasons that Ash mentions? It implies that everyone is in agreement, but that the thing in question has probably not yet been done, and as such makes a good halfway stage between "proposal" and "implementation".

Reply:
I like "resolved" because it covers both outcomes: in favor or against. But maybe we should just use the built-in language of "closed" for this?

Reply:
Thinking on this more, we could update the template to be something like this:

```
How this RFC will be resolved

This RFC will be *accepted* if...
This RFC will be *rejected* if...
```

That gives some space to explicitly address the two outcomes (acceptance / rejection) and also think through how rejection might occur, which helps prevent ambiguous outcomes that don't feel like either. E.g., "This RFC will be considered rejected if it's not accepted within X days and there's no further discussion."